### PR TITLE
Feature popup window

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description 
- Added pop-up window
- Updated Index File
- Updated Style.CSS File
- Update App.js File

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+
+// Popup Window
+
+const open = document.getElementById('open');
+const modal = document.getElementById('modal-container');
+const close = document.getElementById('close');
+
+open.addEventListener('click', () => {
+  modal.classList.add('show');
+});
```
Comment (jasonfelice) [inline app.js]: - [ ] The JavaScript should contain an array of objects that will be used to render data in the popup window.
_Here's what it should look like:_
```javascript
const projects = [{
    id: 1
    name: "Example",
    description: "Adescription.",
    image: "link_to_image.jpg",
    technologies: ["html", "JavaScript", "css"],
    link: "link_to_live_version",
}]
```
- You can use the array filter method to filter the object you'll need to use to render data.

Good luck 😄

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi, 

Good job so far!
There are some issues that you still need to work on before you move to the next project.<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No linter errors✔️ 
Amazing design✔️ 
Followed GitHub flow✔️ 

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js (hunk 1 above)
Comment (IsmailChangezi) [reply app.js]: Thank you for your feedback. I have completed that requirement

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @IsmailChangezi,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**